src/main/java/edu/harvard/iq/dataverse/search/savedsearch/SavedSearchServiceBean.java

```diff
@@ -189,7 +189,10 @@ public JsonObjectBuilder makeLinksForSingleSavedSearch(DataverseRequest dvReq, S
                     hitInfo.add(resultString, "Skipping because dataset " + datasetToLinkTo.getId() + " is already part of the subtree for " + savedSearch.getDefinitionPoint().getAlias());
                 } else if (datasetAncestorAlreadyLinked(savedSearch.getDefinitionPoint(), datasetToLinkTo)) {
                     hitInfo.add(resultString, "FIXME: implement this?");
-                } else {
+                } else if (!datasetToLinkTo.isReleased()) {
```

Comment:
@sekmiller  you said "I left in a seeming inconsistency in which unpublished datasets are not linked but unpublished dataverses are".  I'm a little confused because it looks like a new else statement. Where was that logic before?

Reply:
The logic is in the command, where if the dataset is a draft it throws a command exception. i thought it would be nicer to handle it here and put it in the log instead of putting up the red banner. (especially if you're linking a bunch of objects from a single search)

Reply:
gotcha, that seems reasonable.